lib/ood_core/batch_connect/template.rb

```diff
@@ -116,27 +116,49 @@ def bash_helpers
                   shuf -i ${1}-${2} -n 1
                 }
                 export -f random_number
+                export python_socket_cmd="import socket; socket.socket().connect(('$1',$2))"
```

Comment:
I don't think this actually works as intended. Instead of exporting, just duplicate the code between the two python methods.

```sh
port_used_python() {
  python -c "import socket; socket.socket().connect(('$1',$2))"
}
port_used_python3() {
  python3 -c "import socket; socket.socket().connect(('$1',$2))"
}
```

You can see how this doesn't work:

```
efranz@owens-login01:~$ export python_socket_cmd="print('$1', $2)"
efranz@owens-login01:~$ port_used_python() { python -c "$python_socket_cmd"; }
efranz@owens-login01:~$ port_used_python localhost 8080
('',)
efranz@owens-login01:~$ port_used_python() { python -c "print('$1', $2)"; }
efranz@owens-login01:~$ port_used_python localhost 8080
('localhost', 8080)
```